### ISSUE TITLE
draft-js example Bad Request (400)

### ISSUE THREAD

**Issue by domsew** [microsoft/FluidFramework#4775]
## Describe the bug
Hello, I'm trying to run [Draft.js example](https://github.com/microsoft/FluidExamples/tree/a6a15616a9149c238ba23a93c0eda3f0c1f14f65/draft-js).
I'm getting bad request error (400). In DevTools I see the following message:
> xhr.js:160 GET http://localhost:3000/repos/tinylicious/commits?token=dGlueWxpY2lvdXM%3D&count=1&sha=1604960165754 400 (Bad Request)

### System Info:
Browser: Chrome 87.0.4280.141 (64-bit)
System: Windows 10 1903
Node: v14.4.0
Git: 2.22.0.windows.1
Link to repo refers to commit I used.

### To Reproduce
Follow instructions in README.md.
I used `npm start`.

### Expected behavior
The example should work.

### Logs
<details>
<summary>Logs</summary>
PS D:\Workspaces\FluidExamples\draft-js> npm start

\> @fluid-example/draft-js@0.1.0 start D:\Workspaces\github\FluidExamples\draft-js
\> concurrently "npm:start:server" "npm:start:client"

[start:server] 
[start:server] > @fluid-example/draft-js@0.1.0 start:server D:\Workspaces\github\FluidExamples\draft-js
[start:server] > tinylicious
[start:server] 
[start:client] 
[start:client] > @fluid-example/draft-js@0.1.0 start:client D:\Workspaces\github\FluidExamples\draft-js
[start:client] > webpack serve
[start:client]
[start:server] info: Listening on port 3000 {"label":"winston","timestamp":"2021-01-11T08:27:38.415Z"}
[start:client] i ｢wds｣: Project is running at http://localhost:8080/
[start:client] i ｢wds｣: webpack output is served from undefined
[start:client] i ｢wds｣: Content not from webpack is served from D:\Workspaces\github\FluidExamples\draft-js
[start:server] info: GET /repos/tinylicious/git/refs/heads/1604960165754 400 11.577 ms - 154 {"label":"winston","timestamp":"2021-01-11T08:27:45.608Z"}
[start:server] info: GET /repos/tinylicious/git/refs/heads/1604960165754 400 5.537 ms - 154 {"label":"winston","timestamp":"2021-01-11T08:27:45.634Z"}
[start:client] i ｢wdm｣: Hash: 7cb668d6a0554c7b259f
[start:client] Version: webpack 4.44.2
[start:client] Time: 19622ms
[start:client] Built at: 11.01.2021 09:28:00
[start:client]         Asset       Size  Chunks             Chunk Names
[start:client] app.bundle.js   11.3 MiB     app  [emitted]  app
[start:client]    index.html  652 bytes          [emitted]
[start:client] Entrypoint app = app.bundle.js
[start:client] [0] multi (webpack)-dev-server/client?http://localhost:8080 ./src/app.tsx 40 bytes {app} [built]
[start:client] [./node_modules/react-dom/index.js] 1.33 KiB {app} [built]
[start:client] [./node_modules/react-router-dom/esm/react-router-dom.js] 10.1 KiB {app} [built]
[start:client] [./node_modules/react/index.js] 190 bytes {app} [built]
[start:client] [./node_modules/strip-ansi/index.js] 161 bytes {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/index.js?http://localhost:8080] (webpack)-dev-server/client?http://localhost:8080 4.29 KiB {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.51 KiB {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.53 KiB {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/utils/createSocketUrl.js] (webpack)-dev-server/client/utils/createSocketUrl.js 2.91 KiB {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/utils/log.js] (webpack)-dev-server/client/utils/log.js 964 bytes {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/utils/reloadApp.js] (webpack)-dev-server/client/utils/reloadApp.js 1.59 KiB {app} [built]
[start:client] [./node_modules/webpack-dev-server/client/utils/sendMessage.js] (webpack)-dev-server/client/utils/sendMessage.js 402 bytes {app} [built]
[start:client] [./node_modules/webpack/hot sync ^\.\/log$] (webpack)/hot sync nonrecursive ^\.\/log$ 170 bytes {app} [built]
[start:client] [./src/app.tsx] 2 KiB {app} [built]
[start:client] [./src/pages/index.ts] 165 bytes {app} [built]
[start:client]     + 773 hidden modules
[start:client] Child HtmlWebpackCompiler:
[start:client]                           Asset      Size               Chunks  Chunk Names
[start:client]     __child-HtmlWebpackPlugin_0  4.88 KiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0
[start:client]     Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
[start:client]     [./node_modules/html-webpack-plugin/lib/loader.js!./public/index.html] 903 bytes {HtmlWebpackPlugin_0} [built]
[start:client] i ｢wdm｣: Compiled successfully.
[start:server] info: OPTIONS /repos/tinylicious/commits?token=dGlueWxpY2lvdXM%3D&count=1&sha=1604960165754 204 0.887 ms - 0 {"label":"winston","timestamp":"2021-01-11T08:28:41.300Z"}
[start:server] info: GET /repos/tinylicious/commits?token=dGlueWxpY2lvdXM%3D&count=1&sha=1604960165754 400 9.484 ms - 135 {"label":"winston","timestamp":"2021-01-11T08:28:41.312Z"}
[start:server] info: OPTIONS /deltas/tinylicious/1604960165754?from=0&to=2000 204 0.200 ms - 0 {"label":"winston","timestamp":"2021-01-11T08:28:41.323Z"}
[start:server] info: GET /deltas/tinylicious/1604960165754?from=0&to=2000 200 3.509 ms - 962 {"label":"winston","timestamp":"2021-01-11T08:28:41.330Z"}
[start:server] info: Disconnect of a7baf5c1-577b-40e0-aace-5f7c6bd56198 {"messageMetaData":{"documentId":"1604960165754","tenantId":"tinylicious"},"label":"winston","timestamp":"2021-01-11T08:28:43.420Z"}
[start:server] info: Disconnect of a7baf5c1-577b-40e0-aace-5f7c6bd56198 from room {"messageMetaData":{"documentId":"1604960165754","tenantId":"tinylicious"},"label":"winston","timestamp":"2021-01-11T08:28:43.422Z"}
[start:server] info: Disconnect of d6774a1b-8361-4f5e-9ad3-d63941639a9f {"messageMetaData":{"documentId":"1604960165754","tenantId":"tinylicious"},"label":"winston","timestamp":"2021-01-11T08:28:50.003Z"}
[start:server] info: Disconnect of d6774a1b-8361-4f5e-9ad3-d63941639a9f from room {"messageMetaData":{"documentId":"1604960165754","tenantId":"tinylicious"},"label":"winston","timestamp":"2021-01-11T08:28:50.004Z"}
[start:server] info: OPTIONS /repos/tinylicious/commits?token=dGlueWxpY2lvdXM%3D&count=1&sha=1604960165754 204 0.370 ms - 0 {"label":"winston","timestamp":"2021-01-11T08:28:50.867Z"}     
[start:server] info: GET /repos/tinylicious/commits?token=dGlueWxpY2lvdXM%3D&count=1&sha=1604960165754 400 5.084 ms - 135 {"label":"winston","timestamp":"2021-01-11T08:28:50.881Z"}       
[start:server] info: OPTIONS /deltas/tinylicious/1604960165754?from=0&to=2000 204 0.238 ms - 0 {"label":"winston","timestamp":"2021-01-11T08:28:50.895Z"}
[start:server] info: GET /deltas/tinylicious/1604960165754?from=0&to=2000 200 1.784 ms - - {"label":"winston","timestamp":"2021-01-11T08:28:50.907Z"}
</details>

**Comment by skylerjokiel** [comment]
Hey @domsew , I was trying to get a repo here with a fresh clone and can't seem to see the issue. When you run `npm start` it starts both the server and the client. Whenever the server closes any existing sessions/tabs that were open will no longer be valid if the server is restarted.

Can you try with a fresh branch and re-running install and start to see if the issue still exists.

**Comment by domsew** [comment]
Hi @skylerjokiel, thanks for your reply. I tried a few things.
1. `npm ci`
2. Delete repo folder and clone again.
3. Restart browser.
4. Restart system.
5. Use another browser (Edge, Firefox).
6. Use another directory (another disk).
7. Clear `%temp%` folder and `npm cache clean --force`.
8. Remove `/var/tmp` directory created by `tinylicious`.

Conclusions:
1. When I used a new directory, app started working once. After restart error appears again.
2. During experiments I observed that the app was worked on Edge or Firefox. Another run - error.
3. For now app only works on Edge.

I do not see any correlations. This behaviour is incomprehensible for me. No more ideas.

**Comment by Flavyoo** [comment]
Have you tried clearing the local storage?

**Comment by domsew** [comment]
I updated the example to newest version (`git pull`), then `npm ci` and then I cleared site data in browser. For now I'm able to simultaneously edit text in two separate browsers. I think that clearing local storage was the solution.

I observed some errors in DevTools Console and in terminal logs, but they are not critical probably. For example `DeltaConnectionFailureToConnect `, `Undefined and null keys are not supported`.

If you wish I can submit another issue with details.

Thanks for your help and your attention.
